components/network/nzbget/Makefile

```diff
@@ -0,0 +1,43 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"). You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2021 erwinlem
+#
+
+BUILD_BITS=64
+include ../../../make-rules/shared-macros.mk
```

Comment:
Please add
USE_OPENSSL11=yes
here and re-run gmake REQUIRED_PACKAGES. This should update the dependencies to openssl-1.1. Otherwise the deprecated openssl-1.0 will be used.